lib/matplotlib/markers.py

```
@@ -132,18 +147,56 @@ def get_alt_transform(self):
+    def _set_custom_marker(self, path):
+        verts = path.vertices
+        rescale = np.sqrt(max(np.max(verts[:,0]**2),
+                              np.max(verts[:,1]**2)))
+        self._transform = Affine2D().scale(1.0 / rescale)
+        self._path = path
+    
+    def _set_path_marker(self):
+        self._set_custom_marker(self._marker)
+        
+    def _set_vertices(self):
+        path = Path(verts)
```

Ruff (0.12.2)

Undefined name `verts`

(F821)